pkg/kubectl/cmd/auth/reconcile.go

```diff
@@ -161,7 +160,7 @@ func (o *ReconcileOptions) RunReconcile() error {
 		shallowInfoCopy := *info
 
 		switch t := info.Object.(type) {
-		case *rbac.Role:
+		case *rbacv1.Role:
```

Comment:
this means you can no longer run `kubectl auth reconcile -f <file containing v1beta1 RBAC objects>`